core/platform/email/gae_email_services.py

```diff
@@ -22,6 +22,15 @@
 
 
 def get_incoming_email_address(reply_to_id):
+    """Gets the incoming email address. The client is responsible for recording
+       any audit logs.
+
+    Args:
+      reply_to_id: str. unique id of sender.
+
+    Returns:
+      (string). email address of sender.
```

Comment:
Write return type as- 'str. The email address of the sender.'

Reply:
Can you address this comment also? Thanks!

Reply:
I have made changes. Please review. Thank you.